.github/workflows/go.yml

```diff
@@ -0,0 +1,34 @@
+name: Go
+on:
+  pull_request: {}
+  push:
+    tags:
+    - 'v[0-9]*'
+env:
+  GO111MODULE: on
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/setup-go@v2
+      with:
+        go-version: 1.14
+      id: go
+    - uses: actions/checkout@v2
+    - run: make test
+    - run: make lint
+  release:
+    name: Release to GitHub Releases
+    runs-on: ubuntu-latest
+    needs: test
+    if: startsWith(github.ref, 'refs/tags/')
+    steps:
+    - uses: actions/setup-go@v2
+      with:
+        go-version: 1.14
+      id: go
+    - uses: actions/checkout@v2
+    - run: ./script/release
+      env:
+        GITHUB_TOKEN: ${{ secrets.MACKERELBOT_GITHUB_TOKEN }}
```

Comment:
Do we really need mackerelbot token?
(Unless necessary, using `secrets.GITHUB_TOKEN` seems enough. ref https://github.com/mackerelio-labs/check-aws-cloudwatch-logs-insights/blob/main/.github/workflows/release.yml#L40 )

Reply:
ok, i will drop this env.